### ISSUE TITLE
Changes to the output of the outxml_parser

### ISSUE THREAD

**Issue by janssenhenning** [JuDFTteam/masci-tools#10]
Up till now the output of the outxml_parser tries to mirror the current status of the hardcoded parser in aiida-fleur as closely as possible. Here we collect things that should be added/modified in the output. Feel free to add to this list

- The normal distance output is named `charge_density`. Maybe not the most telling name (alternative `density_convergence` for example)
- The names for the relax output are also not optimal (some are formatted containing the atomtype, maybe a list is more useful); also see https://github.com/JuDFTteam/aiida-fleur/issues/101
- Add some quality criterions of calculations (lostElectrons, ...)
- Some calculation modes are missing for newer fleur versions (Hubbard1, greensfunction, ...)

**Comment by janssenhenning** [comment]
further modifications

- Remove `targetStructureClass` (this has been empty for a long time)
- Adjust some outputs with more fleur modes (e.g. bandgap output only in with mode='hist')

**Comment by janssenhenning** [comment]
Maybe add git describe as a more detailed description of the used version of fleur

**Comment by janssenhenning** [comment]
number of kpoints is missing

**Comment by janssenhenning** [comment]
- Number of MPI/OMP should be available (can we reliably calculate core-h from this ?)

**Comment by Tseplyaev** [comment]
@janssenhenning 
The number of MPI/OMP used in the calculation is available in the attributes of a CalcJob node:
```
In [1]: a = load_node(55639)   #CalcJob node                                                                                                                            

In [2]: a.attributes                                                                                                                                       
Out[2]: 
{'job_id': '17021989',
 'sealed': True,
 'version': {'core': '1.4.0', 'plugin': '1.1.0'},
 'withmpi': True,
 'resources': {'num_machines': 4,
  'num_cores_per_mpiproc': 8,
  'num_mpiprocs_per_machine': 6},
  ...
```
There is also information on execution time and other useful data, related to submission

**Comment by Tseplyaev** [comment]
I think there should be no reason for keeping `force_largest` if forces were not calculated. If I am not mistaken, `force_largest` is used for information only and removing it should not affect functionality of other parts of the plugin.

**Comment by janssenhenning** [comment]
> The number of MPI/OMP used in the calculation is available in the attributes of a CalcJob node:
Okay then the best thing is to leave this to the aiida side

**Comment by janssenhenning** [comment]
@Tseplyaev Would it be okay to convert the relax output to one list? So for example instead of 

```python
{
    'force_x_type1': 1,
    'force_y_type1': 2,
    'force_z_type1': 3,
    'force_x_type2': 4,
    'force_y_type2': 5,
    'force_z_type2': 6,
}
```
We write
```python
{
    'force': [[1,2,3],
              [4,5,6]]
}
```
And the same for `abspos`. We could also call it `force_atoms`.

I would also like to rename `force_largest` to `force_largest_component` (Since that is what's it is :))

**Comment by Tseplyaev** [comment]
@janssenhenning 
Sure, I think making a more compact form of storing the result is better. I believe the force output of the FleurCalculation `output_parameters` is used nowhere, hence there is no compatibilities to worry about.
The name change sounds reasonable, this is indeed the largest component.

**Comment by janssenhenning** [comment]
These changes are now implemented in masci-tools version `0.4.0-dev7`
